Observer/AdminhtmlCustomerSaveAfterObserver.php

```diff
@@ -0,0 +1,834 @@
+     * @param \Magento\MediaStorage\Model\File\UploaderFactory $fileUploaderFactory
+     * @param ProductCollection                                $sellerProduct
+     * @param \Magento\Framework\Json\DecoderInterface         $jsonDecoder
+     * @param MpSalesPartner                                   $mpSalesPartner
+     * @param MpSeller                                         $mpSeller
+     * @param \Magento\Customer\Model\CustomerFactory          $customerFactory
+     * @param MpHelper                                         $mpHelper
+     * @param MpEmailHelper                                    $mpEmailHelper
+     * @param \Webkul\Marketplace\Model\ProductFactory         $mpProductFactory
+     * @param \Magento\Framework\Module\Dir\Reader             $reader
+     * @param Filesystem                   $filesystem
+     * @param ProductModel                                     $productModel
+     */
+    public function __construct(
+        Filesystem $filesystem,
+        \Magento\Framework\Stdlib\DateTime\DateTime $date,
+        \Magento\Framework\Message\ManagerInterface $messageManager,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
+        \Magento\Catalog\Api\ProductRepositoryInterface $productRepository,
+        CollectionFactory $collectionFactory,
+        \Magento\MediaStorage\Model\File\UploaderFactory $fileUploaderFactory,
+        ProductCollection $sellerProduct,
+        \Magento\Framework\Json\DecoderInterface $jsonDecoder,
+        MpSalesPartner $mpSalesPartner,
+        MpSeller $mpSeller,
+        \Magento\Customer\Model\CustomerFactory $customerFactory,
+        MpHelper $mpHelper,
+        MpEmailHelper $mpEmailHelper,
+        \Webkul\Marketplace\Model\ProductFactory $mpProductFactory,
+        \Magento\Framework\Module\Dir\Reader $reader,
+        ProductModel $productModel
+    ) {
+        $this->_mediaDirectory = $filesystem->getDirectoryWrite(DirectoryList::MEDIA);
+        $this->_fileUploaderFactory = $fileUploaderFactory;
+        $this->_productRepository = $productRepository;
+        $this->_messageManager = $messageManager;
+        $this->_collectionFactory = $collectionFactory;
+        $this->_storeManager = $storeManager;
+        $this->_date = $date;
+        $this->_sellerProduct = $sellerProduct;
+        $this->_jsonDecoder = $jsonDecoder;
+        $this->mpSalesPartner = $mpSalesPartner;
+        $this->mpSeller = $mpSeller;
+        $this->customerFactory = $customerFactory;
+        $this->mpHelper = $mpHelper;
+        $this->mpEmailHelper = $mpEmailHelper;
+        $this->mpProductFactory = $mpProductFactory;
+        $this->reader = $reader;
+        $this->filesystem = $filesystem;
+        $this->productModel = $productModel;
+    }
+
+    /**
+     * admin customer save after event handler.
+     *
+     * @param \Magento\Framework\Event\Observer $observer
+     */
+    public function execute(\Magento\Framework\Event\Observer $observer)
```

Comment:
import (use) namespace at the beginning of the file

Reply:
Yes, will setup magento-coding-standard scan and will fix